Assets/Scripts/Items/Item.cs

```diff
@@ -0,0 +1,15 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+namespace AIM.Items
+{
+    public class Item : MonoBehaviour
```

Comment:
This is the base class for all items, right? Then this should be abstract.
```suggestion
    public abstract class Item : MonoBehaviour
```